FAQ.md

```diff
@@ -1,202 +1,122 @@
-> 
-> 4. Overwhelming amounts of content. TOP takes about a year in most cases to make a good developer. Imagine trying to do multiple TOPs and consider the previous points in this consideration.
->
-> 5. Tutorial hell: It's very easy to bounce between resources and just do the beginner stuff over and over and over making it feel like you're solidifying concepts (this is related to point 1), not committing to one course is going to make it a lot easier to bounce between different beginner courses looking for the "best" one. However, there is no "best", when you're a self-learner, it's about learning yourself and not being handed the concepts.
->
-> 6. Communities: Often times it's hard to find help for topics on courses that do not have a good community and it can be rude to bring that baggage to another community.
->
-> 7. If your courses use different languages, you can easily confuse the two and struggle to write decent code in either. Concepts transfer, but only after you're a programmer, not still struggling with basic stuff.
->
-> Oone of the biggest mistakes you can do as a learner is try to follow multiple courses at the same time.
+Nope, this is run by the community, for the community without pay of any sort. 
 
-**Is there an Odin dark theme?**
-Yup! 
+**What is a "help vampire" or "vamp"?** 
 
-https://chrome.google.com/webstore/detail/the-odin-project-dark-the/omiecnmmajomefpodcdjkebeonieonef?hl=en-US
+[This](http://slash7.com/2006/12/22/vampires/) is actually a really good read about one of the biggest problems that exists in online communities. Simply asking questions does not make a help vampire. 
 
-https://addons.mozilla.org/en-US/firefox/addon/the-odin-project-dark-theme/
+**How can I help others solve their coding problems?** 
 
-**Is TOP hard?**
-Very, but that doesn't mean you can't do it. Learning programming in general is tough, no matter where you go. Grit is something you will need to develop to get through it, read about it here: https://en.wikipedia.org/wiki/Grit_(personality_trait)
+Read our [guide on helping others solve coding problems](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/join-the-odin-community#how-to-help-others-solve-coding-problems). 
 
-**Does the TOP staff get paid?**
-Nope, this is run by the community, for the community without pay of any sort.
+**How should I ask questions?**
 
-**Can I promote here?**
-Very likely not, people trying to make money on The Odin Project is not part of our mission. Self promotion permission is granted on an individual basis. If you don't have permission, don't promote. This does not apply to sharing things you've built in the showcase for the purpose of sharing. Promotion for products or services there will be deleted.
+[Read this]([https://discord.com/channels/505093832157691914/505093832157691916/853784735292325898](https://www.theodinproject.com/how_to_ask)).
```

Comment:
```suggestion
[This lesson about asking questions](https://www.theodinproject.com/how_to_ask) in the curriculum is a great starting point.
```

Reply:
This was the one I mentioned but apparently changed the link for incorrectly (fixed it).

The link to discord: https://discord.com/channels/505093832157691914/505093832157691916/853784735292325898

I am not committing my suggestion yet since I want to change the link to the actual lesson in the curriculum instead of the technical questions article currently linked. 

Just want to hear what link others would prefer.

The discord link is not present in the wiki, unless you think that’s not needed since it turned into a lesson.